index.html

```diff
@@ -0,0 +1,203 @@
+                        <div> 
+                            <p class="main-paragraph">Our youth empowerment programmes tackle youth 
+                            unemployment through skills development. We partner
+                             with organisations who run projects that give young people.
+                            </p>
+                        </div>
+                    </div>  
+                    <div class="event">
+                        <div class="sub-event">  
+                             <img src="images/program5.png" class="main-image" alt="program-img5">
+                            <a href="#" class="speaker-content">Discussion</a>
+                        </div>
+                        <div> 
+                            <p class="main-paragraph">Our youth empowerment programmes tackle youth 
+                            unemployment through skills development. We partner
+                             with organisations who run projects that give young people.
+                            </p>
+                        </div>
+                    </div>
+                </div>
+               <a href="#" class="come">See the whole program</a>
+                <button type="submit" class="join">Join Summit 2022</button>
+        </main>
+         <!-- about the speaker session making it dynamic using js-->
+
+    <section class="about-cont" id="join">
+        <h1 class="speak-head">Featured Speakers</h1>
+        <div class="line3"></div>
+        <div class="about-head" id="speakers">
+            </div>
+        </div>
+            <button type="submit" class="btn" onclick="myFunction()">MORE 
+                <i class="fa fa-angle-down" aria-hidden="true"></i>
+            </button>
+        
+    </section>
+
+        <!-- partner section start -->
+    
+        <section id="sponsors">
+        <div class="foot-cont">
+            <div class="foot-head">
+                <h1 class="footer-heading">Partners</h1>
+                <div class="foot-box">
+                    <img src="images/coca-cola-logo-svgrepo-com.svg" alt="partner1" class="part">
+                    <img src="images/instagram-logo-svgrepo-com.svg" alt="partner2" class="part">
+                    <img src="images/forbes-logo-svgrepo-com.svg" alt="partner3" class="part">
+                    <img src="images/google-play-download-android-app-logo-svgrepo-com.svg" alt="partner4" class="part">
+                    <img src="images/heineken-14-logo-svgrepo-com.svg" alt="partner5" class="part">
+                </div>
+            </div>
+        </div>
+    </section>
+     
+    <!-- footer  -->
+ 
+    <footer class="footer" id="news">
+    <div class="footer-cont">
+      <div class="footer-box">
+        <img src="images/my-logo.png" alt="footer logo"/>
+        <p>Global Youth Summit 2023.</p>
+      </div>
+
+      <div class="footer-text">
+        <p>@ 2023 Global Youth Empowerment
+        </p>
+      </div>
+    </div>
+  </footer>
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you remove the horizontal bar at the bottom of your page as it doesn't give the user a nice experience